config/config_mgmt.py

```diff
@@ -0,0 +1,680 @@
+            print("getDefaultConfig Failed, Error: {}".format(str(e)))
+            raise e
+
+        return defConfigOut
+
+    def _updateDiffConfigDB(self):
+        try:
+            # Get the Diff
+            print('Generate Final Config to write in DB')
+            configDBdiff = self._diffJson()
+            # Process diff and create Config which can be updated in Config DB
+            configToLoad = self._createConfigToLoad(configDBdiff, \
+                self.configdbJsonIn, self.configdbJsonOut)
+
+        except Exception as e:
+            print("Config Diff Generation failed")
+            print(e)
+            raise e
+
+        return configToLoad
+
+    """
+    Create the config to write in Config DB from json diff
+    diff: diff in input config and output config.
+    inp: input config before delete/add ports.
+    outp: output config after delete/add ports.
+    """
+    def _createConfigToLoad(self, diff, inp, outp):
+        ### Internal Functions ###
+        """
+        Handle deletes in diff dict
+        """
+        def _deleteHandler(diff, inp, outp, config):
+            # if output is dict, delete keys from config
+            if isinstance(inp, dict):
+                for key in diff:
+                    #print(key)
+                    # make sure keys from diff are present in inp but not in outp
+                    # then delete it.
+                    if key in inp and key not in outp:
+                        # assign key to None(null), redis will delete entire key
+                        config[key] = None
+                    else:
+                        # log such keys
+                        print("Diff: Probably wrong key: {}".format(key))
+
+            elif isinstance(inp, list):
+                # just take list from output
+                # print("Delete from List: {} {} {}".format(inp, outp, list))
+                #print(type(config))
+                config.extend(outp)
+
+            return
+
+        """
+        Handle inserts in diff dict
+        """
+        def _insertHandler(diff, inp, outp, config):
+            # if outp is a dict
+            if isinstance(outp, dict):
+                for key in diff:
+                    #print(key)
+                    # make sure keys are only in outp
+                    if key not in inp and key in outp:
+                        # assign key in config same as outp
+                        config[key] = outp[key]
+                    else:
+                        # log such keys
+                        print("Diff: Probably wrong key: {}".format(key))
+
+            elif isinstance(outp, list):
+                # just take list from output
+                # print("Delete from List: {} {} {}".format(inp, outp, list))
+                config.extend(outp)
+
+            return
+
+        """
+        Recursively iterate diff to generate config to write in configDB
+        """
+        def _recurCreateConfig(diff, inp, outp, config):
```

Comment:
add docstring to describe the parameter?

Returns the sum of two decimal numbers in binary digits.

	Parameters:
		a (int): A decimal integer
		b (int): Another decimal integer

	Returns:
		binary_sum (str): Binary string of the sum of a and b